chat-commands.js

```diff
@@ -2479,6 +2479,58 @@ exports.commands = {
 	},
 	blacklisthelp: [`/blacklist [username], [reason] - Blacklists the user from the room you are in for a year. Requires: # & ~`],
 
+	battleban: function (target, room, user, connection) {
+		if (!target) return this.parse(`/help battleban`);
+
+		target = this.splitTarget(target);
```

Comment:
It'd be more readable if this was a separate variable; I suggest ``reason``

Reply:
I suppose that are other commands that define it as target although :P